packages/manfred2html/src/engine/cv-monochrome-force/html-parts/experience-section/experience-section.ejs

```diff
@@ -17,7 +17,7 @@
         </div>
         <%_}-%>
         <%_ if(experience.url && experience.url !== "undefined") { -%>
```

Comment:
remove && experience.url !== "undefined", please

Reply:
It's fixed